### PR TITLE
linux-kernel: update to 6.10.13

### PR DESCRIPTION
**Pull request by KexyBiscuit**
Topic Description
-----------------

- linux-kernel: bump rel
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- Revert "linux-kernel: build non-DRM simple framebuffer driver into 64-bit LoongArch kernel"
    This reverts commit 7621d24430541c5b666828724cf56c3745998026.
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- linux-kernel: bump rel
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- linux-kernel: build non-DRM simple framebuffer driver into 64-bit LoongArch kernel
    Reported-by: 安慕希
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- Revert "linux-kernel: build simple framebuffer driver into 64-bit LoongArch kernel"
    This reverts commit ec8d46242a76a41ebc24ddd779ed45ddf619c58f.
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- linux-kernel: build simple framebuffer driver into 64-bit LoongArch kernel
    Reported-by: 李克思
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- kernel-tools: update to 6.10.13
    - Synchronize patches from linux-kernel.
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- linux-kernel: update to 6.10.13
    - Track patches at https://github.com/AOSC-Tracking/linux @ aosc/v6.10.13, current HEAD is 1e5c70e4641692ec5226794187d12ef339c83989.
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- linux-kernel: update Power ISA config for 6.10.13-rc1
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
- kernel-tools: update to 6.10.13-rc1
    - Synchronize patches from linux-kernel.
    Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>

... and 2 more commits

Package(s) Affected
-------------------

- kernel-tools: 6.10.13-1
- linux+kernel: 3:6.10.13
- linux-kernel-${__VER}: 1:6.10.13-4

Security Update?
----------------

No

Build Order
-----------

```
#buildit linux-kernel linux+kernel kernel-tools
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by KexyBiscuit** [comment]
Smoke testing passed on AMD64 generation 2, AMD64 generation 1 and Armv8-A generation 2 Hyper-V VMs.

image
image
image

**Comment by KexyBiscuit** [comment]
Smoke testing passed on AMD64 VMware VM.

- Product: VMware® Workstation 17 Pro
- Version: 17.6.0 build-24238078
- Host OS version: Windows 11 Enterprise, 64-bit (Build 27718.1000) 10.0.27718
- Accelerate 3D graphics: on
- Firmware type: UEFI
- Host VBS Mode: on

image

**Comment by KexyBiscuit** [comment]
Smoke testing passed on Latitude D630.

图片

**Comment by KexyBiscuit** [comment]
Smoke testing passed on 战神Z8-KP7S2.

图片
图片

**Comment by KexyBiscuit** [comment]
Smoke testing passed on RedmiBook 16 (锐龙版).

图片

**Comment by KexyBiscuit** [comment]
Smoke testing passed on Surface Pro (5th Gen).

图片

**Comment by KexyBiscuit** [comment]
Smoke testing passed on ROG Ally (2023).

图片

**Comment by KexyBiscuit** [comment]
Smoke testing passed on ProArt B550-CREATOR.

图片

**Comment by KexyBiscuit** [comment]
Smoke testing passed on AMD64 VMware VM with 6.10.13.

File: runtime-kernel/linux-kernel/autobuild/ppc64el/config
Comment (MingcongBai) [inline runtime-kernel/linux-kernel/autobuild/ppc64el/config:1]: Hmm, we sure about this?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by MingcongBai** [comment]
Tested good on LoongArch (THTF Chaorui L680-T2).

File: runtime-kernel/linux-kernel/autobuild/ppc64el/config
Comment (KexyBiscuit) [reply runtime-kernel/linux-kernel/autobuild/ppc64el/config:1]: In https://github.com/torvalds/linux/commit/44ac4625ea002deecd0c227336c95b724206c698, the config is marked as broken.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**